src/app/videos/[id]/page.tsx

```diff
@@ -0,0 +1,104 @@
+"use client"
+import { useEffect, useState } from "react"
+import axios from "axios"
+import { useRouter } from "next/navigation"
+import { usePathname, useSearchParams } from "next/navigation"
+import Image from "next/image"
+import YoutubeLogo from "/public/images/logos/YouTube-White-Dark-Background-Logo.wine.svg"
+
+import { VideoItem } from "../../../../utils/static/fetchVideosFromYouTubers"
+
+const VideoPage = () => {
+  const router = useRouter()
+  const pathname = usePathname()
+  const searchParams = useSearchParams()
+
+  const idSegment = pathname.split("/").pop()
```

Comment:
is this a query param or? maybe we can get this param easier than

Reply:
In case of fetching data https://spotify23.p.rapidapi.com/episode/?id=${id} it is being used as one, but when you look at my  local app , for example http://localhost:3000/podcasts/2rXPZ6QAo0Iu8JwZhTw2vI, it's part of the URL path not a query param though.